pkg/network/ebpf/c/http-types.h

```diff
@@ -52,6 +52,7 @@ typedef struct {
     __u16 response_status_code;
     __u64 response_last_seen;
     char request_fragment[HTTP_BUFFER_SIZE] __attribute__ ((aligned (8)));
+    size_t fragment_sz;
```

Comment:
`fragment_len` might be a bit easier to understand versus knowing that `sz = size`.